build-ps/pxc_builder.sh

```diff
@@ -468,7 +474,7 @@ install_deps() {
         apt-get -y install doxygen doxygen-gui graphviz rsync libcurl4-openssl-dev
         apt-get -y install libcurl4-openssl-dev libre2-dev pkg-config libtirpc-dev libev-dev
         apt-get -y install --download-only percona-xtrabackup-24=2.4.29-1.${DIST}
```

Comment:
Have we got pxb-24 arm versions?

Reply:
@adivinho Yes, I've added pxb-24 arm builds for debian/ubuntu before building pxc-80 arm builds